.gitignore

```diff
@@ -1,10 +1,12 @@
-interface/bower_components/
+*.swn
+*.swo
```

Comment:
what's a .swn file?

Reply:
It's a type of swapfile, also with swo. My own computer doesn't make them, but it would seem that @movingelectrons and presumably others do. I would think it's from some text editor or other like *.tmp.